src/neptune/metadata_containers/metadata_container.py

```diff
@@ -653,15 +654,40 @@ def _shutdown_hook(self):
         self.stop()
 
     def _fetch_entries(
-        self, child_type: ContainerType, query: NQLQuery, columns: Optional[Iterable[str]], limit: Optional[int]
+        self,
+        child_type: ContainerType,
+        query: NQLQuery,
+        columns: Optional[Iterable[str]],
+        limit: Optional[int],
+        sort_by: str,
     ) -> Table:
         if columns is not None:
-            # always return entries with `sys/id` column when filter applied
+            # always return entries with 'sys/id' and the column chosen for sorting when filter applied
             columns = set(columns)
             columns.add("sys/id")
+            columns.add(sort_by)
+
+        if sort_by == "sys/creation_time":
+            sort_by_column_type = AttributeType.DATETIME
+        else:
+            sort_by_column_type = _get_atomic_column_type(
+                backend=self._backend,
+                project_id=self._project_id,
+                child_type=child_type,
+                column=sort_by,
+            )
+
+            if not not _validate_atomic_column_type(sort_by_column_type):
```

Comment:
Something weird happened here with ‘not not’

Reply:
![image](https://github.com/neptune-ai/neptune-client/assets/58885668/e0cfbdc3-4079-499d-980a-21fd993a6ecd)

Reply:
Fortunately it was only introduced by the last commit, so when the e2e tests ran a few commits back it was actually ok